x-pack/plugins/apm/public/components/shared/Links/apm/transaction_detail_link.tsx

```diff
@@ -23,6 +24,8 @@ interface Props extends APMLinkExtendProps {
   transactionType: string;
   latencyAggregationType?: string;
   environment?: string;
+  comparisonEnabled?: boolean;
```

Comment:
typo?
```suggestion
  comparisonEnabled?: toBooleanRt;
```

Reply:
not really, this is the typescript definition not the io-ts.

Reply:
Ahh, I misread!